<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertAceToSqliteDialog</class>
 <widget class="QDialog" name="ConvertAceToSqliteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import ACE File</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblSource">
        <property name="minimumSize">
         <size>
          <width>105</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Source URL:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leSource">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="cbAddToProject">
      <property name="text">
       <string>Add to project</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblDest">
        <property name="minimumSize">
         <size>
          <width>105</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Destination URL:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leDest"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="tbDest">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>98</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConvertAceToSqliteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConvertAceToSqliteDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
